content/en/docs/started/workstation/kubeflow-on-microk8s.md

```diff
@@ -20,26 +22,25 @@ To get Kubeflow running using MicroK8s, you'll need to install MicroK8s, enable
 
 **Note:** You need MicroK8s version 1.18 and above to enable and run Kubeflow.
 
-1. Install MicroK8s with [Snap](https://snapcraft.io/) by running the following command:
```

Comment:
This was added, I think, in case someone didn't have Snap installed (and skipped one of the previous paragraphs). I'd recommend leaving this here ("with [Snap](...)...")

Reply:
A lot of information is probably not read but scanned by users, depending on the cognitive workload.

Reply:
The point here was to not direct users away from the docs, and snapcraft.io is a store, so it will not really provide great help, I think that if users don't have snaps they will be able to see it in the command and find how to get snaps to work on their Linux distro